### PR TITLE
CDAP-1998: Remove deprecated Context - Query param in Metrics v3 API

### PR DESCRIPTION
**Pull request by abaranau**

### CODE REVIEW

File: cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java [hunk 1]
```diff
@@ -273,33 +253,11 @@ private void tagsQuerying(HttpRequest request, HttpResponder responder, List<Str
...
-    }
-  }
-
   private MetricQueryResult executeQuery(HttpRequest request, Map<String, String> sliceByTags,
-                                         List<String> groupByTags, List<String> metrics) {
-    try {
-      MetricQueryRequest queryRequest = new MetricQueryRequest(sliceByTags, metrics, groupByTags);
-      setTimeRangeInQueryRequest(queryRequest, new QueryStringDecoder(request.getUri()).getParameters());
-      return executeQuery(queryRequest);
-    } catch (IllegalArgumentException e) {
```
Comment (gsps1) [inline cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java]: we don't want to differentiate between invalid request and internal_server_error ?

Comment (abaranau) [inline cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java]: it is old code. It is handled one level up now

Comment (abaranau) [inline cdap-watchdog/src/main/java/co/cask/cdap/metrics/query/MetricsHandler.java]: I.e. it *was* not differentiating, now it is

**Comment by gsps1** [comment]
LGTM

**Comment by abaranau** [comment]
http://builds.cask.co/browse/CDAP-DUT1946-2

**Comment by abaranau** [comment]
http://builds.cask.co/browse/CDAP-DUT1946-11 GREEN

**Pull request merged**